templates/pre_buildspec.yml.tpl

```diff
@@ -9,11 +9,17 @@ phases:
       - BUILD_CONDITION=$(cat ci.txt)
       - PR_NUMBER=$(cat pr.txt)
       - SRC_CHANGED=$(cat src_changed.txt)
+      - |
+        if [ "${PIPELINE_TYPE}" == "cd"];then
```

Comment:
Approved, just before the merge please update this condition (addition of service_changed condition).